tools/cldr-apps/src/main/java/org/unicode/cldr/web/VettingViewerQueue.java

```diff
@@ -263,44 +264,36 @@ public boolean isStopped() {
             }
         }
 
-        Thread myThread = null;
-        public boolean stop = false;
+        private Thread myThread = null;
+        private boolean stop = false;
 
-        public CLDRLocale locale;
+        private CLDRLocale locale;
         private QueueEntry entry;
-        SurveyMain sm;
-        VettingViewer<Organization> vv;
-        public int maxn;
-        public int n = 0;
-        final public boolean isSummary;
-        public long start = -1;
-        public long last;
-        public long rem = -1;
+        private SurveyMain sm;
+        private int maxn;
+        private int n = 0;
+        private final boolean isSummary;
+        private long start = -1;
+        private long last;
+        private long rem = -1;
         private final String st_org;
-        final Level usersLevel;
-        final Organization usersOrg;
-        String status = "(Waiting my spot in line)";
-        public Status statusCode = Status.WAITING; // Need to start out as
+        private final Level usersLevel;
+        private final Organization usersOrg;
+        private String status = "(Waiting my spot in line)";
+        private Status statusCode = Status.WAITING; // Need to start out as waiting.
 
-        // waiting.
-
-        void setStatus(String status) {
+        private void setStatus(String status) {
             this.status = status;
         }
 
-        public float progress() {
-            if (maxn <= 0)
-                return (float) 0.0;
-            return ((float) n) / ((float) maxn);
-        }
-
-        StringBuffer aBuffer = new StringBuffer();
+        private StringBuffer aBuffer = new StringBuffer();
 
-        public Task(QueueEntry entry, CLDRLocale locale, SurveyMain sm, String baseUrl, Level usersLevel,
+        private Task(QueueEntry entry, CLDRLocale locale, SurveyMain sm, String baseUrl, Level usersLevel,
             Organization usersOrg, final String st_org) {
             isSummary = isSummary(locale);
-            if (DEBUG)
-                System.err.println("Creating task " + locale.toString());
+            if (DEBUG) {
+                System.out.println("Creating task " + locale.toString());
```

Comment:
better just `logger.debug(…)` (and drop `DEBUG`) but this is fine for now

Reply:
i had a hunch you would suggest logger... however DEBUG is currently defined as DEBUG = false || CldrUtility.getProperty("TEST", false) ... not sure if that functionality is still desired and didn't want to experiment with it right now ...